h/views/api/decorators/client_errors.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from pyramid.httpexceptions import HTTPForbidden, HTTPNotFound
+
+
+def client_error_decorator(wrapped):
+    """
+    Decorate client (HTTP 4xx) exception views
```

Comment:
I was wondering about the name of this decorator and the first line of its docstring. "Decorate client error views" seems quite generic and describes where the decorator is used not what it does.

I wonder if it might be better to name the decorator after what it does. And that might also suggest splitting this into two separate decorators that each do one thing. Something like `@forbidden_view_config(..., decorator=(cors_policy, change_fordidden_to_notfound, change_notfound_to_unsupported_media_type)):`

Reply:
This is a really good idea. I'm going to split them up.